.github/workflows/build.yml

```
@@ -0,0 +1,122 @@
+      uses: codecov/codecov-action@v3
+      with:
+        file: ./coverage/lcov.info
+        fail_ci_if_error: false
+
+    - name: Build application
+      run: npm run build
+      env:
+        NODE_ENV: production
+
+    - name: Run Lighthouse CI
+      uses: treosh/lighthouse-ci-action@v10
```

actionlint (1.7.7)

the runner of "treosh/lighthouse-ci-action@v10" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)